src/lib.rs

```diff
@@ -238,6 +238,12 @@ impl EventsLoop {
             events_loop_proxy: self.events_loop.create_proxy(),
         }
     }
+
+    /// Makes it possible for glutin to register a callback when a suspend event happens on Android
+    #[cfg(target_os = "android")]
+    pub fn set_suspend_callback(&self, cb: Option<Box<Fn(bool) -> ()>>) {
```

Comment:
This method should be in the `os::android` module to be consistent with other similar methods.

Reply:
Done